packages/toolpad-app/src/components/EditableText.tsx

```diff
@@ -121,6 +129,7 @@ const EditableText = React.forwardRef<HTMLInputElement, EditableTextProps>(
             whiteSpace: 'nowrap',
             fontSize: theme.typography[typographyVariant].fontSize,
             height: theme.typography[typographyVariant].fontSize,
```

Comment:
Isn't the cropping due to the height set to the fontsize here?

Reply:
😄 Good catch!